addon/utils/store.js

```diff
@@ -21,11 +21,30 @@ export default EmberObject.extend({
   },
 
   initialsFor(properties) {
-    let key = hash(properties);
```

Comment:
Can we just write a polyfill on that function without changing the public interface implementation?
I mean, instead of `_ensureHashable` just write our own `_hash` method which will replace the object-hash module.

Reply:
Sure, np.